app/views/results/edit.html.erb

```diff
@@ -164,6 +168,27 @@
   </aside>
 <% end %>
 
+<!-- Progress Bar -->
+<div class="progress_meter">
+    <span id="progress_span">
+    </span>
+</div>
+
+<script type="text/javascript">
+    <%
+      if @current_user.admin?
+        num_marked = @assignment.get_num_marked
+        num_assigned = @assignment.get_num_assigned
+      else
+        num_marked = @assignment.get_num_marked(@current_user.id)
+        num_assigned = @assignment.get_num_assigned(@current_user.id)
+      end
+    %>
+    $( document ).ready(function() {
+        update_bar(<%= num_marked %>, <%= num_assigned %>);
+    });
+</script>
```

Comment:
Better, but please merge this with the other `$(document).ready` callback earlier in this file.